### PR TITLE
Added nikto tool completion

### PR DESCRIPTION
**Pull request by tbhaxor**
## Description

Added completion for the Nikto (web vulnerability testing tool)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] If I have added a new file, I also added it to ``clean_files.txt`` and formatted it using ``lint_clean_files.sh``.
- [ ] I have added tests to cover my changes, and all the new and existing tests pass.

### CODE REVIEW

**Review by gaelicWizard — APPROVED** [review]

**Comment by tbhaxor** [comment]
@NoahGorny Ping 😅

File: completion/available/nikto.completion.bash [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# shellcheck shell=bash
+
+__nikto_completion() {
+	local cur=${COMP_WORDS[COMP_CWORD]}
```
Comment (gaelicWizard) [inline completion/available/nikto.completion.bash:4]: This line isn't used anymore, so you can remove it. `shellcheck` is complaining about the unused variable.
```suggestion
```

**Review by gaelicWizard — CHANGES_REQUESTED** [review]

File: completion/available/nikto.completion.bash (hunk 1 above)
Comment (tbhaxor) [reply completion/available/nikto.completion.bash]: @gaelicWizard how did you proposed this change. This is amazing 🤩 

image

**Review by gaelicWizard — APPROVED** [review]
LGTM

**Comment by davidpfarrell** [comment]
@tbhaxor  did you write this yourself?  I can't find any existing completions online.

Either way, I think it would better for you to host this completion in its own repository and vendor it into bash-it.

That way, the project can evolve on its own and even be used by non-bash-it users.

I think we need to get away from owning completions in the bash-it source tree ...

**Comment by tbhaxor** [comment]
Got it. @davidpfarrell will try this weekend

**Comment by davidpfarrell** [comment]
> Got it. @davidpfarrell will try this weekend

Awesome !

Once the completion is up in its own repo, I bet you could get it patched into either the official bash-completion package or just into the nikto homebrew formula.

I always like when I install a brew app and it it brings along its completions.

A prime example of this is the `tmux` brew formula which I installed yesterday and comes with its own completion.  Turns out this completion is hosted by a 3rd party repo, here's the brew formula for it:

```
  resource "completion" do
    url "https://raw.githubusercontent.com/imomaliev/tmux-bash-completion/f5d53239f7658f8e8fbaf02535cc369009c436d6/completions/tmux"
    sha256 "b5f7bbd78f9790026bbff16fc6e3fe4070d067f58f943e156bd1a8c3c99f6a6f"
  end
```

This shows the power of giving completions their own repository ... 


side note: We probably don't need to host a version of the tmux completion in bash-it any longer - I may do a review and a pr to remove it ...

**Pull request closed without merging**